autoscaler/README.md

```diff
@@ -0,0 +1,37 @@
+# Horizontal Self Scaler container
+
+This container image watches over the number of schedulable nodes in the cluster and resizes
+the number of replicas in the required controller.
+
+Usage of pod_nanny:
+    --configmap <params>
+    --rc <replication-controller>
+    --rs <replica set>
+    --deployment <deployment>
+    --verbose
+    --namespace <namespace>
+```
+
+# Implementation Details
+
+The code in this module is a Kubernetes Golang API client that, using the default service account credentials
+available to Golang clients running inside pods, it connects to the API server and polls for the number of nodes
```

Comment:
Yes, @thockin commented above about the same - we need two scale maps, one for cores and one for nodes, and we lookup both maps and choose the greater number. The user may choose to omit one map (and just scale only by number of cores or nodes). I am changing it to accept the two scale maps.